final-project/manRoomStates.tex

```diff
@@ -0,0 +1,21 @@
+\chapter{Maintaining Room State \\
+  \small{\textit{-- Evan Ciok, Sophia DiCuffa, Carson McManus}}
+  \index{Chapter!room-state}
+  \label{Chapter::RoomState}}
+
+\section{Lost Balancer-Monolith Connections}
+
+The connection between the load balancer and the individual monoliths is vital to the operation of OTT, so when one of these connections is lost, it needs to be handled appropriately. 
+The most important consideration is what to do with all clients connected to rooms on that given monolith. Without the balancer connecting to it, it is no longer viable and all clients 
+should be unloaded from the rooms. From there, they can reconnect to a room through the load balancer to a monolith with a valid connection. 
+
+\section{Duplicate Rooms Across Monoliths}
```

Comment:
It would also be really good to have a diagram for this section.

Reply:
what type of diagram do you think? sequence?

Reply:
Sure, sounds good.